README.md

```diff
@@ -49,6 +49,13 @@ Clicking the "Stop" button in the debugger will kill the Bazel process being
 debugger, allowing you to halt the current build. The Bazel server, however,
 will continue running.
 
+## Using the LSP (Experimental)
+
+This extension can use [Facebook's starlark project](https://github.com/facebookexperimental/starlark-rust) as a language server.
+
+1. Install the LSP using cargo: `cargo install starlark_bin`
```

Comment:
We also need to add the binary path to `PATH`.

Reply:
`cargo install` will put this in `~/.cargo/bin`, and this should be added to your path during installation of cargo.